BigInteger/BigInteger.cs

```diff
@@ -5,6 +5,8 @@ public class BigInteger
     // maximum length of the BigInteger in uint (4 bytes)
     // change this to suit the required level of precision.
     private const int maxLength = 70;
+    private const int MAX_LENGTH = 70;
```

Comment:
MAX_LENGTH is there to replace maxLength. All constants' name should follow conventions to increase code readability.